src/config.ini

```diff
@@ -27,6 +27,7 @@ num_course_metadata=10000
 num_classes=12000
 num_slots=32000
 num_slot_professor=55000
+num_slot_class=32000
```

Comment:
Just asking what these numbers are

Reply:
These numbers are used in the scrapper progress bars to determine the percentage of the progress.
However, since they are hardcoded, it is a rough approximation

Reply:
uhh, I see